packages/Material-ToolBuilder.package/MDPluggableList.class/instance/basicKeyPressed..st

```diff
@@ -1,25 +1,18 @@
 keystrokes
 basicKeyPressed: aChar 
 
-	| milliseconds slowKeyStroke listSize newSelectionIndex oldSelectionIndex |
-	oldSelectionIndex := newSelectionIndex := self getCurrentSelectionIndex.
-	listSize := self getListSize.
-	listSize = 0 ifTrue: [^self flash].
+	| milliseconds slowKeyStroke |
+
+	self getListSize = 0 ifTrue: [^ self flash].
 	milliseconds := Time millisecondClockValue.
 	slowKeyStroke := (Time
 		milliseconds: milliseconds
 		since: self lastKeystrokeTime) > (self filterableList ifTrue: [500] ifFalse: [300]).
 	self lastKeystrokeTime: milliseconds.
 	slowKeyStroke
-		ifTrue:
-			[ self filterableList ifTrue: ["self hasFilter ifFalse: [ self priorSelection: (self modelIndexFor: self selectionIndex)]" ].
-			"forget previous keystrokes and search in following elements"
-			self lastKeystrokes: aChar asLowercase asString.
-			newSelectionIndex := newSelectionIndex \\ listSize + 1.
-			"self filterableList ifTrue: [ list := self listCollection ]" ]
+		ifTrue: [self lastKeystrokes: aChar asLowercase asString]
 		ifFalse: [ "append quick keystrokes but don't move selection if it still matches"
```

Comment:
Meiner Meinung nach kann der Kommentar weg, sodass dann Zeile 15 mit dem ifFalse: in eine Zeile kann.